cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -910,7 +918,7 @@ impl<I: VCodeInst> VCode<I> {
                             }
                         }
 
-                        if self.insts[iix.index()].is_move().is_some() {
+                        if self.is_move.contains_key(&iix) {
```

Comment:
Reusing the cached decision about what instruction is a move meant that we keep the move instructions that involve fixed non-allocatable registers in the final assembly.